vision/unit_tests/test_coordinate_lengths.py

```diff
@@ -0,0 +1,151 @@
+"""
+Testing vision.deskew.coordinate_lengths.py
+"""
+
+import unittest
+
+import numpy as np
+
+from vision.deskew.coordinate_lengths import latitude_length
+from vision.deskew.coordinate_lengths import longitude_length
+
+
+class TestCoordinates(unittest.TestCase):
+    """
+    Testing the coordinate lengths module
+    """
+
+    # Testing latitude_length
+
+    # Tests the maximum possible latitude in degrees as an input to the latitude_length function
+    def test_latitude_length_at_max_val(self) -> None:
+        """
+        Asserts that latitude_length returns expected value at maximum input (90 degrees)
+
+        Returns
+        -------
+        None
+        """
+        max_input_in_degrees = 90
+        max_input_in_radians: float = np.deg2rad(max_input_in_degrees)
+        latitude_length_at_max_latitude: float = (
+            111132.92
+            - 559.82 * np.cos(2 * max_input_in_radians)
+            + 1.175 * np.cos(4 * max_input_in_radians)
+            - 0.0023 * np.cos(6 * max_input_in_radians)
+        )
+        function_return_at_max_latitude = latitude_length(max_input_in_degrees)
```

Comment:
Similar comment about the length of your variable names. You are being a bit too unnecessarily descriptive with them. These 4 variables for instance could be renamed as follows:
- `max_latitude_deg` (To match `latitude_length()`'s parameter name)
- `max_latitude_rad`
- `expected_latitude_length`
- `resulting_latitude_length`

Makes it a bit less wordy and better matches the purpose of the variables.